fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentUtils.scala

```diff
@@ -144,7 +144,9 @@ object FaciaContentUtils {
   )
   def maybeShortUrl(fc: FaciaContent) = fieldsGet(fc)(_.flatMap(_.shortUrl))
   def shortUrl(fc: FaciaContent): String = maybeShortUrl(fc).getOrElse("")
-  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replaceFirst("^[a-zA-Z]+://gu.com", ""))
+  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(
+    _.replaceFirst("^https?://gu.com", "")
```

Comment:
I don't think it should be necessary to keep the gu.com removal should it?

Reply:
Yeah, I was keeping it in for safety, but I think that if CAPI has fully moved over to the longer domain now it should be safe to remove it.